package.json

```
@@ -31,7 +35,11 @@
     "jscodeshift": "^0.13.0",
     "lint-staged": "^12.3.5",
     "prettier": "^2.5.1",
-    "turbo": "latest"
+    "prop-types": "^15.8.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "turbo": "latest",
+    "typescript": "^4.6.2"
```

```suggestion
    "prop-types": "^15.8.1",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "turbo": "latest",
    "typescript": "^4.6.2"
```
Also these (except `turbo`)